src/mlpack/methods/naive_bayes/naive_bayes_classifier_impl.hpp

```diff
@@ -162,8 +162,7 @@ void NaiveBayesClassifier<MatType>::Train(const VecType& point,
 }
 
 template<typename MatType>
-void NaiveBayesClassifier<MatType>::Classify(const MatType& data,
-                                             arma::Row<size_t>& results)
+void NaiveBayesClassifier<MatType>::Classify(const MatType& data, arma::Row<size_t>& results, MatType& results_proba)
```

Comment:
This line is longer than 80 characters, and you shouldn't use underscores in variable names.  (These comments apply throughout the PR.)  Take a look at the style guidelines and please make sure your code conforms: https://github.com/mlpack/mlpack/wiki/DesignGuidelines#style-guidelines